### DISCUSSION TITLE
對藝術與性別的關係有興趣嗎?

### DISCUSSION THREAD

**Discussion by Chattenoire** [1101-GS4719-C/Python-Community-GS4719-C#2182] [Show and tell]
這裡推薦一本書，順便放上前陣子為它寫的書評!

《觀看的方式》
觀看是什麼？體會事物與我們的聯繫，而非見其本身。

過去事物在我們觀看時，總朦朧著一層神秘性——它們被建築好的規則牽引、解讀；文化不再擁有可重現的過去，而是繼承著無數個當下的過去。如今，這層面紗透過資本在所有人的面前重現，而過去則成為遙望者眼中的聖地，不可褻瀆、只屬權貴，是為統治。

這是個關於油畫的故事。現在，你應能理解文物不再只是神聖遺產或展覽旁的歷史故事所形塑的作品，而是創作自身。然而，創作本身暗示了什麼？

從女性裸畫作為景觀/男性作為觀賞者，到男性如蛇般引誘女性吃下鏡子的毒果，使自身亦為共犯。這一切反過來說明了「原初」的神聖性：原始部落/非歐洲傳統中無性別的性熱情，超越了墮落的文明歐洲人。

文藝復興式的男性傳統懼怕成為景觀，而也許現代的境況便由此始；油畫沒有消失，而是化身廣告，播送著一幅幅的女性標準、男性教條。

然而，畫家是否如此不堪？非，但傳統之所以無形難脫，便是因其會重新將反叛/革新者，納入自身麾下，再次形成傳統；彷彿，藝術家的反抗，諷刺地助其成長為龐然大物。

如今，現代的油畫帶來什麼？不同於油畫帶來的實體力量，廣告提供的，是虛幻的選擇與白日夢。

線上購買書籍連結: https://www.books.com.tw/products/0010895449

**Comment by Chattenoire** [comment]
自己原創的書評，結果這樣被判定是AI寫的，到底是稱讚還是貶低阿@@

**Reply by zoo100130** [reply]
好壞，我也來寫寫看，看會不會被判定
